scio-core/src/main/java/com/spotify/scio/transforms/FileDownloadDoFn.java

```diff
@@ -62,15 +62,16 @@ public FileDownloadDoFn(RemoteFileUtil remoteFileUtil, SerializableFunction<Path
    * @param keep keep downloaded files after processing.
    */
   public FileDownloadDoFn(
-      RemoteFileUtil remoteFileUtil,
+      PipelineOptions options,
       SerializableFunction<Path, OutputT> fn,
       int batchSize,
       boolean keep) {
-    this.remoteFileUtil = remoteFileUtil;
     this.fn = fn;
     this.batch = new ArrayList<>();
     this.batchSize = batchSize;
     this.keep = keep;
+
+    RemoteFileUtil.configure(options);
```

Comment:
Aren't we able to access the pipeline option in the `@Setup` method to run this code ?
We wouldn't need the options in the constructor